app/utils/mail.py

```
@@ -1,15 +1,21 @@
 """
-Email utility module for LeagueLedger using FastAPI-Mail
+Email utility module for LeagueLedger using Python's built-in email and smtplib packages
 """
 import os
+import smtplib
+import asyncio
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.utils import formatdate
 from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)